package.json

```diff
@@ -37,14 +43,30 @@
 		"definitions-clean": "rimraf dist/node_modules/vscode-dev-containers",
 		"definitions-copy": "copyfiles --all \"node_modules/vscode-dev-containers/**/*\" dist",
 		"npm-pack": "npm pack",
+		"pkg-pack": "cross-env PKG_CACHE_PATH=node_modules/.pkg-cache pkg -c package.json -o dist/devcontainer devcontainer.js",
+		"pkg-pack-linux-x64": "cross-env PKG_CACHE_PATH=node_modules/.pkg-cache pkg -c package.json -o dist/devcontainer-linux-x64 -t linux-x64 devcontainer.js",
+		"pkg-pack-linux-arm64": "cross-env PKG_CACHE_PATH=node_modules/.pkg-cache pkg -c package.json -o dist/devcontainer-linux-arm64 -t linux-arm64 devcontainer.js",
+		"pkg-pack-win-x64": "cross-env PKG_CACHE_PATH=node_modules/.pkg-cache pkg -c package.json -o dist/devcontainer-win-x64 -t win-x64 devcontainer.js",
+		"pkg-pack-win-arm64": "cross-env PKG_CACHE_PATH=node_modules/.pkg-cache pkg -c package.json -o dist/devcontainer-win-arm64 -t win-arm64 devcontainer.js",
 		"clean": "npm-run-all clean-dist clean-built",
 		"clean-dist": "rimraf dist",
 		"clean-built": "rimraf built",
-		"test": "env TS_NODE_PROJECT=src/test/tsconfig.json mocha -r ts-node/register --exit src/test/*.test.ts",
-		"test-matrix": "env TS_NODE_PROJECT=src/test/tsconfig.json mocha -r ts-node/register --exit",
+		"test": "env TS_NODE_PROJECT=src/test/tsconfig.json mocha -r ts-node/register --exit --exclude '**/node_modules/**/*.ts' 'src/test/**/*.test.ts'",
+		"test-matrix": "env TS_NODE_PROJECT=src/test/tsconfig.json mocha -r ts-node/register --exit --exclude '**/node_modules/**/*.ts'",
 		"test-container-features": "env TS_NODE_PROJECT=src/test/tsconfig.json mocha -r ts-node/register --exit src/test/container-features/*.test.ts",
 		"test-container-templates": "env TS_NODE_PROJECT=src/test/tsconfig.json mocha -r ts-node/register --exit src/test/container-templates/*.test.ts"
 	},
+	"pkg": {
+		"assets": [
+			"scripts/updateUID.Dockerfile"
```

Comment:
The output when running with `--debug` suggests that it includes a lot of files from `node_modules` when these are already packaged using esbuild. Can we exclude these?

Also add `dist/node_modules` to the assets, we will remove the in the future, currently we still need them.

Reply:
I can check on the `node_module` packaging. I assume `@vercel/pkg` is including them as dependencies of `node-pyt`, but maybe it's copying the entire dir. I'll add dist/node_modules`.